NEWS.md

```diff
@@ -1,5 +1,9 @@
 # glue (development version)
 
+* `+` now works in more situations, and gives errors when one side isn't a 
```

Comment:
This is a breaking change, but I doubt that many people relied on the fact that `glue("{x}") + "{y}"` would also interpolate the RHS.

Reply:
Ha! I just learned that this was featured in the README 🙈